IDEAS/Buildings/Components/Window.mo

```diff
@@ -182,15 +197,24 @@ protected
     annotation (Placement(transformation(extent={{-20,-12},{0,-32}})));
   Fluid.Sources.OutsideAir       outsideAir(
     redeclare package Medium = Medium,
-    Cs=Cs,
-    Habs=Habs, azi = aziInt,
+    Cs=if not Use_custom_Cs and sim.interZonalAirFlowType == IDEAS.BoundaryConditions.Types.InterZonalAirFlow.TwoPorts
+         then sim.Cs_coeff*Habs 
```

Comment:
`then sim.Cs_coeff*Habs `

Should become:

`then sim.Cs_coeff*(Habs^(2*sim.a))`